src/js/subscribe-input.js

```diff
@@ -4,42 +4,131 @@ function validateEmail(email) {
   return email === '' ? true : re.test(email);
 }
 
-function initInput() {
-  const $input = document.querySelector('.field input');
-  const $label = document.querySelector('.field label');
+function initRegisterForm($form) {
+  const $doneHeader = $form.querySelector('.form-done h4');
+  const $doneButton = $form.querySelector('.form-done .button');
+  const $submitButton = $form.querySelector('.form-submit .button');
+  const $input = $form.querySelector('.field input');
+  const $label = $form.querySelector('.field label');
 
-  if ($input) {
-    function validateField(ev = {}) {
-      const isActive = $input === document.activeElement || $input.value !== '' || ev.type === 'focus';
+  function validateField(ev = {}) {
+    const isActive = $input === document.activeElement || $input.value !== '' || ev.type === 'focus';
 
-      $input.parentNode.parentNode.classList.toggle('is-active', isActive);
+    $input.parentNode.parentNode.classList.toggle('is-active', isActive);
 
-      if (isActive) {
-        const isValidationMessageActive  = !validateEmail($input.value);
+    if (isActive) {
+      const isValidationMessageActive = !validateEmail($input.value);
 
-        $input.parentNode.parentNode.classList.toggle('is-invalid', isValidationMessageActive);
+      $input.parentNode.parentNode.classList.toggle('is-invalid', isValidationMessageActive);
 
-      } else {
-        $input.parentNode.parentNode.classList.toggle('is-invalid', false);
-      }
+      return isValidationMessageActive;
+    } else {
+      $input.parentNode.parentNode.classList.toggle('is-invalid', false);
     }
+  }
 
-    function makeInputActive() {
-      $input.parentNode.parentNode.classList.add('is-active');
-      $input.focus();
-    }
+  function makeInputActive() {
+    $input.parentNode.parentNode.classList.add('is-active');
+    $input.focus();
+  }
+
+  if ($doneHeader && $doneButton && $input && $submitButton) {
```

Comment:
I would suggest to clear input after user press done. So it would be empty again.

Reply:
It is cleared in 58th line